src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -1133,6 +1133,21 @@ namespace InternalHeatGains {
 						Lights( Loop ).FractionReturnAirIsCalculated = ( AlphaName( 6 ) == "YES" );
 					}
 
+					// Set return air node number
+					Lights( Loop ).ReturnNodePtr = 0;
+					std::string retNodeName = "";
+					if ( !lAlphaFieldBlanks( 7 ) ) {
+						if ( LightsObjects( Item ).ZoneListActive ) {
+							ShowSevereError( RoutineName + CurrentModuleObject + "=\"" + Lights( Loop ).Name + "\": " + cAlphaFieldNames( 7 ) + " must be blank when using a ZoneList." );
+							ErrorsFound = true;
+						} else {
+							retNodeName = AlphaName( 7 );
+						}
+					}
+					if ( Lights( Loop ).ZonePtr > 0 ) {
+						Lights( Loop ).ReturnNodePtr = DataZoneEquipment::GetReturnAirNodeForZone( Zone( Lights( Loop ).ZonePtr ).Name, retNodeName );
```

Comment:
So ReturnNodePtr will equal 0 if there is no return used by HVAC? How will the lights know NOT to put heat gain to return when there is no HVAC connection?